### PR TITLE
Fix the bug in time format culture-affected

### PR DESCRIPTION
**Pull request by LowEntropyBody**
Related issue: #4655

### CODE REVIEW

**Comment by yufeih** [comment]
I'm thinking about setting `CultureInfo.CurrentCulture` and `CultureInfo.CurrentUICulture` globally to invariant culture at program startup, because there would be a lot of places where we call `ToString` without specifying invariant culture, and docfx as a tool doesn't really have much culture dependent text.

File: src/docfx/cli/Docfx.cs [hunk 1]
```diff
@@ -62,6 +63,9 @@ internal static async Task<int> Run(string[] args)
                 return 1;
             }
 
+            CultureInfo.CurrentCulture = new CultureInfo("en-US", false);
```
Comment (yufeih) [inline src/docfx/cli/Docfx.cs]:
```suggestion
            CultureInfo.CurrentCulture = CultureInfo.CurrentUICulture = new CultureInfo("en-US", false);
```

Comment (LowEntropyBody) [reply src/docfx/cli/Docfx.cs]: Cool

File: src/docfx/cli/Docfx.cs [hunk 2]
```diff
@@ -62,6 +63,8 @@ internal static async Task<int> Run(string[] args)
                 return 1;
             }
 
+            CultureInfo.CurrentCulture = CultureInfo.CurrentUICulture = new CultureInfo("en-US", false);
```
Comment (yufeih) [inline src/docfx/cli/Docfx.cs]: what does `false` mean here, is `new CultureInfo("en-US")` enough?

Comment (LowEntropyBody) [reply src/docfx/cli/Docfx.cs]: You are right

**Pull request merged**